libfreeipmi/driver/ipmi-inteldcmi-driver.c

```diff
@@ -444,6 +445,19 @@ ipmi_inteldcmi_ctx_io_init (ipmi_inteldcmi_ctx_t ctx)
       goto cleanup;
     }
 
+  flags = fcntl(ctx->device_fd, F_GETFD);
+  if (flags == -1)
+    {
```

Comment:
minor nit, could we do < 0 throughout this patch? It's the style in the rest of FreeIPMI.